site/content/entry/yuzu-progress-report-mar-2020/index.md

```diff
@@ -0,0 +1,131 @@
+
+{{< imgs
+    "./11.png| Incorrect."
+    "./12.png| Much better."
+  >}}
+
+We also have a cool improvement by [Tobi](https://github.com/FearlessTobi).
+The Switch is a surround sound compatible console and our implementation of downmixing removed any channel that wasn’t left or right.
+By keeping the extra channels and mixing them in the left and right stereo outputs, the sound in tons of games, including Luigi’s Mansion 3, Sonic Forces, and Bayonetta, has been improved.
+
+## A peculiar case of "Even a broken clock is right twice a day!"
+
+The Nintendo Switch supports `ASTC`, a type of texture format generally used in mobile hardware such as phones and tablets in order to reduce bandwidth usage (a limited resource in those devices).
+Since it's a heavily compressed format, decoding it via the CPU, like it is normally done on PC, is a time-consuming task.
+
+To give an example, Astral Chain is a game that uses `ASTC` very extensively and in bizarre ways, as previous fixes and optimizations have shown.
+On NVIDIA and AMD graphics hardware, the first scene of a new game normally takes typically `17` seconds to load. The GPU has to wait for the CPU to decode all the `ASTC` textures first, thus, slowing down the whole process.
+
+Thanks to [Rodrigo's](https://github.com/ReinUsesLisp) research, it was discovered that on any recent Intel GPU this same operation is immediate and occurs with no delay, thanks to it being the only vendor providing a native `ASTC` decoder in their hardware.
+That’s right, the best way to experience Astral Chain in yuzu is with a small, and typically inferior, iGPU (like the ones that come in any desktop or notebook CPU from Intel).
+
+On the topic of formats, implementing the `RG32` and `RGB32` vertex formats fixed a room that refused to load in The Legend of Zelda: Link’s Awakening in Vulkan.
+Great work [Rodrigo](https://github.com/ReinUsesLisp)!
+
+{{< imgs
+    "./13.png| Works!"
+  >}}
+  
+## Paving the road for even bigger changes
+
+[Rodrigo](https://github.com/ReinUsesLisp)'s initial work on Transform Feedback has concluded and is now partially working. 
+This already helps games like POKKÉN Tournament DX, Donkey Kong Country: Tropical Freeze and Xenoblades Chronicles.
+
+An optimization by [Blinkhawk](https://github.com/FernandoS27) reduced the use of RAM by 1GB.
+When talking about RAM usage, less is best!
+
+Now, an important but also astonishing fix; one that nobody expected, not even our project leader [bunnei](https://github.com/bunnei).
+While he was working on fixing what seemed to be a simple bug on Final Fantasy VIII, he also accidentally fixed Super Smash Bros. Ultimate fights!
+This case of serendipity is without a doubt a surprise, but a welcome one.
+
+Now all users can brawl all they want thanks to a measly 5-line change in yuzu’s code!
```

Comment:
Using `all` back to back reads poorly. Perhaps change the 2nd `all they want` to `to their hearts' content`